cogeo_mosaic/create.py

```diff
@@ -15,41 +14,37 @@ def create_mosaic(
     dataset_list: Sequence[str],
     minzoom: int = None,
     maxzoom: int = None,
-    quadkey_zoom: int = None,
     max_threads: int = 20,
-    minimum_tile_cover: float = None,
-    tile_cover_sort: bool = False,
     version: str = "0.0.2",
     quiet: bool = True,
+    **kwargs,
 ) -> Dict:
     """
     Create mosaic definition content.
 
     Attributes
     ----------
-    dataset_list : tuple or list, required
-        Dataset urls.
-    minzoom: int, optional
-        Force mosaic min-zoom.
-    maxzoom: int, optional
-        Force mosaic max-zoom.
-    quadkey_zoom: int, optional
-        Force mosaic quadkey zoom.
-    minimum_tile_cover: float, optional (default: 0)
-        Filter files with low tile intersection coverage.
-    tile_cover_sort: bool, optional (default: None)
-        Sort intersecting files by coverage.
-    max_threads : int
-        Max threads to use (default: 20).
-    version: str, optional
-        mosaicJSON definition version
-    quiet: bool, optional (default: True)
-        Mask processing steps.
+        dataset_list : tuple or list, required
+            Dataset urls.
+        minzoom: int, optional
+            Force mosaic min-zoom.
+        maxzoom: int, optional
+            Force mosaic max-zoom.
+        minimum_tile_cover: float, optional (default: 0)
+            Filter files with low tile intersection coverage.
+        tile_cover_sort: bool, optional (default: None)
+            Sort intersecting files by coverage.
+        max_threads : int
+            Max threads to use (default: 20).
+        version: str, optional
+            mosaicJSON definition version
+        quiet: bool, optional (default: True)
+            Mask processing steps.
 
     Returns
     -------
-    mosaic_definition : dict
-        Mosaic definition.
+        mosaic_definition : dict
+            Mosaic definition.
```

Comment:
I though I reverted the additional padding @kylebarron

Reply:
Maybe that was a merge conflict